corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -504,36 +505,6 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                 return 'commcare-user';
             };
 
-            if (self.case_preload) {
```

Comment:
what happens to this?

Reply:
Pretty sure this is a weird diff, but if you look at the full diff and still have a question, let me know.